### PR TITLE
Warn about potential crash in "Getting Started"

### PR DESCRIPTION
**Pull request by austinried**
Same changes as  #1479 but updated for the new docs format.

### CODE REVIEW

File: docs/docs/basics/getting-started.md [hunk 1]
```diff
@@ -4,19 +4,8 @@ sidebar_position: 2
...
-
-await TrackPlayer.setupPlayer({})
-// The player is ready to be used
-```
-
-You also need to register a [playback service](./playback-service.md) right after
-registering the main component of your app:
-
+## Starting off
+First, you need to register a [playback service](#playback-service) right after registering the main component of your app (typically in your `index.js` file at the root of your project):
```
Comment (jspizziri) [inline docs/docs/basics/getting-started.md:8]: I believe we should be pointing at `./playback-service.md` instead of `#playback-service`.

**Review by jspizziri — CHANGES_REQUESTED** [review]

File: docs/docs/basics/getting-started.md (hunk 1 above)
Comment (austinried) [reply docs/docs/basics/getting-started.md:8]: Ah forgot about that, I'll switch it back

**Comment by jspizziri** [comment]
@austinried this lgtm. My only other thought is that technically this applies to 2.1 too which would be updated in the versioned_docs folder.Would you be able to do that too?

**Comment by austinried** [comment]
@jspizziri no problem, just pushed that change.

**Review by jspizziri — APPROVED** [review]
@dcvz LGTM. You'll need to merge.

**Pull request merged**